.github/workflows/ci-pypi-deploy.yml

```diff
@@ -0,0 +1,24 @@
+name: package-release
+
+on:
+  workflow_dispatch:
+  pull_request:
+  push:
+    branches:
+     - main
+  release:
+    types:
+      - published
+
+jobs:
+  build:
+    name: build and upload release to pypi
+    runs-on: ubuntu-latest
+    steps:
+      - uses: actions/checkout@v2
+      - uses: actions/setup-python@v2
+      - uses: casperdcl/deploy-pypi@v2
```

Comment:
Can we not get away with the more common github action [pypi-publish](https://github.com/marketplace/actions/pypi-publish)? pypi-publish has a lot more maintainers and a larger community making it more likely to provide updates or use `pdm publish`?

Reply:
ok, yes we can, I am just used to using this action and I am bit loathe change this since I won't know until we create the release whether I have made a typo. Unless you have already used `pypi-publish` and have an example I can copy? ☺️

Reply:
Understand. I recently have used this [pdm publish](https://github.com/observingClouds/ecmwfspec/blob/main/.github/workflows/release.yml)

Reply:
@leifdenby we can use the action for now to move this forward

Reply:
Let's stay with the current implementation